packages/core/src/subgraph/queries/community.ts

```diff
@@ -144,3 +144,25 @@ export const getCommunityUBIParams = async (
         throw new Error(error);
     }
 };
+
+export const communityEntities = async (where: string, fields: string) => {
+    try {
+        const query = gql`
+            {
+                communityEntities(
+                    ${where}
+                ) {
+                    ${fields}
+                }
+            }
+        `;
+
+        const queryResult = await client.query({
+            query,
```

Comment:
learned from https://github.com/impactMarket/backend/pull/223
```suggestion
            query,
            fetchPolicy: 'no-cache',
```